lib/utilshare/accounts/dwolla.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Utilshare.Accounts.Dwolla do
+
+  def token do
+    GenServer.call(Utilshare.TokenManager, {:get, :token})
+  end
+
+  def authenticate_server() do
+    body = {:form, 
+    [
+      client_id: Application.get_env(:utilshare, :key),
+      client_secret: Application.get_env(:utilshare, :secret),
+      grant_type: "client_credentials",
+    ]}
+    case HTTPoison.post("https://sandbox.dwolla.com/oauth/v2/token", body, %{"Content-type" => "application/x-www-form-urlencoded"}) do
+      {:ok, %HTTPoison.Response{status_code: 200, body: body}} ->
+        auth = Poison.decode!(body)
+      {:error, %HTTPoison.Error{reason: reason}} ->
+        IO.inspect(reason)
+    end
+  end
+  #From HttpPoison Tests
+  #https://github.com/edgurgel/httpoison/blob/master/test/httpoison_test.exs
+  def get_header(headers, key) do
+    headers
+    |> Enum.filter(fn({k, _}) -> k == key end)
+    |> List.first
+    |> elem(1)
+  end
+
+  def create_customer(user) do
+    #Creates a verified user
+    headers = ["Authorization": "Bearer "<>token,
+               "Content-Type": "application/vnd.dwolla.v1.hal+json",
+               "Accept": "application/vnd.dwolla.v1.hal+json"]
+    user = %{ firstName: user["first"], lastName: user["last"], email: user["email"] }
+    {:ok, body} = Poison.encode(user)
+    IO.puts body
+    case HTTPoison.post("https://api-sandbox.dwolla.com/customers",body,headers)do
```

Comment:
We should probably store `"https://api-sandbox.dwolla.com/"` as a global constant somewhere. `config/*.exs`?